mne_bids/path.py

```diff
@@ -94,19 +107,18 @@ def entities(self):
             ('acquisition', self.acquisition),
             ('run', self.run),
             ('processing', self.processing),
-            ('recording', self.recording),
             ('space', self.space),
-            ('prefix', self.prefix),
-            ('suffix', self.suffix)
+            ('recording', self.recording),
+            ('kind', self.kind),
         ])
 
-    def __str__(self):
-        """Return the string representation of the path."""
+    @property
+    def basename(self):
```

Comment:
wonderful. Very explicit. Just make sure to document it in the class docstring (I think it should be done)